.travis.yml

```diff
@@ -21,7 +20,7 @@ env:
 install:
     - |
       MINICONDA_URL="http://repo.continuum.io/miniconda"
-      MINICONDA_FILE="Miniconda3-3.7.3-MacOSX-x86_64.sh"
+      MINICONDA_FILE="Miniconda3-latest-MacOSX-x86_64.sh"
```

Comment:
Yes. The version of miniconda was being overridden anyway, so we may as well pull the latest miniconda rather than sitting on that old version.